tutorials/en/vectara-beginner-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-use-case.mdx

```diff
@@ -0,0 +1,402 @@
+---
+
+Title: "Vectara Beginner App Tutorial: Showcase The Creation Of Vectara App In Legal Use Case"
+Description: "Embark on a step-by-step journey to create an intuitive Vectara application specifically tailored for the legal domain. Uncover the nuances of integrating semantic search and user-friendly interfaces, and get a glimpse into the streamlined process of providing automated legal consultations."
+Author Username: "sanchayt743"
+---
+
+## 1. Introduction to the Vectara Ecosystem: 🌌
+
+Welcome to the realm of Vectara, a platform where search transcends to new horizons powered by Generative AI. Here, I will unravel the essence of the Vectara ecosystem, its fundamental workings, and provide a visual journey through Vectara's official materials to enhance your comprehension.
+
+### 🚀 Overview of Vectara Ecosystem
+
+Vectara is on a mission to redefine search, facilitating a seamless journey from a query to the most relevant information. The platform harbors a complete, but composable search pipeline, making it a powerhouse of semantic search capabilities. Through Vectara, developers are empowered to create applications with a robust search backbone, thus elevating the user experience to a realm where questions meet precise answers.
+
+### 🛠 Fundamental Workings and Workflow
```

Comment:
A few comments here:
1. Again like above I would emphasize search with summarization (although one can use just search, summary is optional, but with RAG it's the main focus point).
2. I would remove the second step (data indexing). With Vectara, from the user point of view, when you "ingest" the data, it's sent to the Vectara backend and indexed there. So maybe somehow combine #1 and #2?

Reply:
Can you explain what do you mean by the second point?